### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by ntthang-dev-zz** [wakatime/vscode-wakatime#186]
- Issue Type: `Bug`
- Extension Name: `vscode-wakatime`
- Extension Version: `5.0.1`
- OS Version: `Windows_NT x64 10.0.19041`
- VS Code version: `1.55.1`

:warning: We have written the needed data into your clipboard. Please paste! :warning:

**Comment by alanhamlett** [comment]
@ntthang-dev Please add the traceback to this issue.

**Comment by ntthang-dev** [comment]
Thank you for your interest and help! Sorry I have described the problem not specifically. The WakaTime extension I use on VSCode seems to have an impact on the performance of VSCode as well as the usage process, during use, there are times when WakaTime does not really track the exact time that I have. implement well as this extension doesn't work sometimes. And I got a message from my own VSCode that this issue is quite impacting so I have reported this issue. 😄
